### ISSUE TITLE
crash-stats.mozilla.org no longer follows symbolic links to identify module filenames

### ISSUE THREAD

**Issue by karlt** [rust-minidump/rust-minidump#971]
e.g. https://crash-stats.mozilla.org/report/index/4ffc6620-fdee-4ab2-b541-a2c140240220#tab-modules
Expected: Filename or Debug Filename libexpat.so.1.8.10.
Actual: Filename and Debug Filename both libexpat.so.1.

Version information is still kind of available in the Debug Identifier, but this requires guessing the library version, finding a library built with exactly the same compiler, options, and dependencies, and running dump_syms on the library. 

Canonical full paths (/usr/lib64/libexpat.so.1.8.10) are in the .dmp files, so this appears to be a regression in their processing, though others think that is [less clear](https://bugzilla.mozilla.org/show_bug.cgi?id=1883294#c2).  It regressed sometime before bug 1824634 comment 3.

Changes for https://bugzilla.mozilla.org/show_bug.cgi?id=1847098 may mitigate this considerably, but I wonder whether they are making some assumptions about filename formats.

**Comment by gabrielesvelto** [comment]
@karlt this has been fixed in bug 1847098, now the version field in the module list on Socorro correctly reports the version number as it appears in the file path (versus the SONAME which is encoded in the minidump). See [this crash](https://crash-stats.mozilla.org/report/index/b9b49ef4-72b6-4bc0-923b-1ef4a0250212#tab-modules) for example: glib is reported as `libglib-2.0.so.0` and the version number is 0.8200.4.0 and indeed the path to the file is `/usr/lib/libglib-2.0.so.0.8200.4`

Shall we close this issue?

**Comment by karlt** [comment]
Yes, thanks for your help with this. I've used the info already.
